third_party/nvfuser/csrc/kernel_db/test/test_nvfuser_kernel_db_write.cpp

```diff
@@ -0,0 +1,77 @@
+#if defined(USE_CUDA)
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <torch/torch.h>
+
+#include <kernel_db/kernel_db.h>
+#include <kernel_db/utils.h>
+#include <test/test_gpu_validator.h>
+#include <test/test_utils.h>
+
+// Tests go in torch::jit
+namespace torch {
+namespace jit {
+
+namespace Nvf = torch::jit::fuser::cuda;
+
+// RUN CMD: bin/test_jit --gtest_filter="NVFuserTest*KernelDb_Write*"
+TEST_F(NVFuserTest, KernelDb_Write_CUDA) {
+  // Setup the test db
+  fs::path test_data =
+      fs::path(__FILE__).parent_path() / "test_data/kernel_db_for_query_test";
```

Comment:
I think these two files `kernel_0.cubin` & `kernel_0.cu` should be moved to `kernel_db_for_write_test`, and so is this folder name.